resources/aws/bucket_object.go

```diff
@@ -0,0 +1,55 @@
+package aws
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+type BucketObject struct {
+	AWSEntity
+	Bucket *Bucket
+	Data   string
+	Name   string
+}
+
+// CreateIfNotExists is not implemeted because S3 overwrites bucket objects in
+// case of name clashes. This means that a newer CloudConfig with the same name
+// as an old one will always overwrite it.
+// This shouldn't be a problem, since we use the hash of the CloudConfig
```

Comment:
But we're not using the hash yet are we? At the moment we use the master or worker prefix. Just wanted to check but I think we're OK with overwriting for now.